### ISSUE TITLE
Difficult to select multiple options in dropdown lists

### ISSUE THREAD

**Issue by bryjbrown** [ldbase/ldbase_issues#2]
Some form elements have very lengthy lists of options in their dropdown lists, and using CTRL to select multiple options isn't very user friendly. It would be nice to have a different type of selection process for this that makes sorting through the options and selecting multiple options more usable.

**Comment by bryjbrown** [comment]
Dev team is looking into options for this and will provide demos of different options at a later date.

**Comment by barsenault-fsu** [comment]
Enabled the Choices.js select option and removed the ctrl-click text in https://github.com/ldbase/ldbase_config/commit/83d77540c3e3f0d5696baf899154f74b89e639a0 

Will demo it and await confirmation that the psych team accepts this as a fix.

**Comment by bryjbrown** [comment]
Currently waiting on @Favenzio to implement styling changes as well, once styling is in place and the psych team accepts this as a fix, this issue can be resolved.

**Comment by WilhelminavanDijk** [comment]
Psych team likes the pill fix. It works fine.
We currently don't have an opinion about the styling considering that this may change as we move forward with the design elements.

**Comment by Favenzio** [comment]
Pills have been styled. Since Psych team is ok with the functionality, I'm closing this issue.
